apps/site/assets/css/_footer.scss

```diff
@@ -58,38 +139,303 @@
+      display: block;
+    }
+    @include media-breakpoint-up(lg){
+      display: block;
+
+    }
+  }
+  @include media-breakpoint-up(lg) {
+    margin-bottom: 0px;
+    padding-right: 40px;
+    font-size: $font-size-base;
+    .container {
+      ul {
+        display: flex;
+        flex-direction: column;
+        .m-footer__group {
+          width: 100%;
+        }
+      }
+    }
+
+    .m-footer__group {
+      padding-left: 15px;
+    }
   }
 
-  .m-footer--mobile__items {
-    text-align: center;
+  :nth-child(2) {
+    div {
+      @include media-breakpoint-up(lg) {
+        padding-left: 0px;
+      }
+    }
+    :first-child {
+      padding-top: 0px;
+    }
+    :nth-child(1) {
+      margin-top: 0;
+      margin-bottom: 0px;
+    }
+  }
 
-    @include not-last-child {
-      &::after {
-        color: $white;
-        content: '●';
-        margin-left: .25em;
-        margin-right: .25em;
+  .m-menu__feature_phone {
+    padding-top: 1rem;
+    a {
+      color: $brand-primary;
+    }
+    @include media-breakpoint-down(xs) {
+      display: flex;
+      a {
+        margin-top: 0px;
+        font-size: 14px;
+        display: inline;
+      }
+      strong {
+        padding-right: 4px;
+        font-size: 14px;
+        display: inline;
       }
     }
   }
+  @include media-breakpoint-up(lg) {
+    a {
+      padding-top: 12px;
+      margin-bottom: 0px;
+      display: inline;
+    }
+  }
+}
+
+.call-us-header {
+  @include media-breakpoint-up(lg) {
+    font-size:24px;
+    margin-top: 18px;
+    margin-bottom: 12px;
+    font-family: Inter UI,Helvetica Neue,Helvetica,Arial,sans-serif;
+  }
+  @include media-breakpoint-down(md) {
+    padding-left: 12px;
+  }
+  @include media-breakpoint-down(xs) {
+    margin-top: 2px;
+    font-size: 18px;
+    margin-bottom: 0px;
+  }
+}
+
+.numbers-container {
+  padding-top: 19px;
+  .m-footer__item {
+    display: flex;
+  }
+  strong {
+    margin-top: 12px;
+    padding-right: 4px;
+  }
+  a {
+    color: $brand-primary;
+  }
+}
+
+.all-container {
+  max-width: 100%;
+  @include media-breakpoint-up(lg) {
+    display: flex;
+  }
+}
 
-  .m-footer--mobile__item {
+.lower-footer {
+  @include media-breakpoint-down(xs) {
+    flex-direction: column;
+  }
+
+  @include media-breakpoint-up(lg) {
+    position: absolute;
+    margin: 0 40px 0 56px;
+    margin-right: 80px;
+    left: 0;
+    right: 0;
+    bottom: 60px;
+  }
+
+  display: flex;
+  justify-content: space-between;
+  border-top: 2px solid $brand-primary-light-contrast;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  bottom: 90px;
+
+  .m-menu__language {
+    color: $brand-primary-light-contrast;
+    .custom-language-selector {
+      color: $brand-primary-light-contrast;
+    }
+    
+    .header-globe {
+      display: none;
+    }
+    .footer-globe {
+      display: block;
+    }
+  }
+  
+}
+
+.social-icons-container {
+  @include media-breakpoint-down(xs) {
+    padding-top: 20px;
```

Comment:
Looks like this needs some left and right padding too: 
<img width="353" alt="image" src="https://user-images.githubusercontent.com/2136286/189412208-e0385f7c-ca71-42bb-b75c-20762f5ad64d.png">